consoleApp/src/main/java/com/infoshare/UserRegistration.java

```diff
@@ -0,0 +1,133 @@
+    private static final String REGISTRATION_HELP_TYPE = "W czym możesz pomóc innym, wybierz z listy: ";
+    private static final String REGISTRATION_ERROR = "***BŁĄD***";
+
+    private String getVolunteerName(){
+        String username = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_NAME);
+            Scanner sc = new Scanner(System.in);
+            username = sc.nextLine();
+            if(username.length() > 2){
+                isGood = true;
+            }else{
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+
+        return username;
+    }
+
+    private String getLocation(){
+        String location = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_LOCATION);
+            Scanner sc = new Scanner(System.in);
+            location = sc.nextLine();
+            if(isAlpha(location) == true){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return location;
+    }
+
+    private String getPhoneNumber(){
+        String phone = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_PHONE_NUMBER);
+            Scanner sc = new Scanner(System.in);
+            phone = sc.nextLine();
+            if(isNumber(phone) == true){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return phone;
+
+    }
+
+    private String getEmail(){
+        String email = "";
+        boolean isGood = false;
+        do{
+            System.out.println(REGISTRATION_EMAIL);
+            Scanner sc = new Scanner(System.in);
+            email = sc.nextLine();
+            if(isEmailValid(email) == true ){
+                isGood = true;
+            }else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+        return email;
+    }
+
+    private String getHelpType(){
+        int helpType = 0;
+        boolean isGood = false;
+        TypeOfHelp[] typeOfHelpArray = TypeOfHelp.values();
+        do {
+            System.out.println(REGISTRATION_HELP_TYPE);
+            for (int i = 0; i < typeOfHelpArray.length; i++) {
+                System.out.println((i + 1) + " - " + typeOfHelpArray[i].getTypeOfHelp());
+            }
+            Scanner sc = new Scanner(System.in);
+            helpType = sc.nextInt();
+
+            if (helpType > 0 && helpType < typeOfHelpArray.length + 1) {
+                isGood = true;
+                helpType = helpType - 1;
+            } else {
+                System.out.println(REGISTRATION_ERROR);
+            }
+        }while (isGood == false);
+
+        return typeOfHelpArray[helpType].getTypeOfHelp();
```

Comment:
Wolontariusz może wybrać tylko jedną kategorię pomocy?

Reply:
Na tę chwilę zostawiłam tylko jeden rodzaj pomocy. Wydaje mi się że żeby wybrać więcej niż jeden rodzaj pomocy to pole TypeOfHelp w klasie Volunteer powinno być tablicą, ale na razie nie wiem jak sobie z tym poradzić